src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/jpa/entity/ResettlementAssessmentEntity.kt

```diff
@@ -69,6 +69,12 @@ data class ResettlementAssessmentEntity(
 
   @Column(name = "submission_date")
   var submissionDate: LocalDateTime?,
+
+  @Column(name = "declaration_by")
```

Comment:
I think we can find out this information by using `created_by_user_id` and `creationDate`. We could use a boolean (defaulting to null) called something like `user_declaration`?